app.js

```diff
@@ -2,65 +2,54 @@ const express = require('express');
 const bodyParser = require('body-parser');
 const mongoose = require('mongoose');
 const session = require('express-session');
-// we have to require the module but also call it with the session argument so that this middleware can access the sessions
 const MongoStore = require('connect-mongo')(session);
-const app = express(); 
-
-  
+const app = express();
+const uuidV4 = require('uuid/v4'); //generates cryptographically strong secret with UUID-v4
 const mongoStore = require('express-mongoose-store')(session, mongoose);
-mongoose.connect('mongodb://localhost:27017/proto_2'); //connection or existing mongoose connection
-
 const db = mongoose.connection;
-// mongo error
+
+mongoose.connect('mongodb://localhost:27017/proto_2'); //connection or existing mongoose connection
 db.on('error', console.error.bind(console, 'connection error:'));
 
-const uuidV4 = require('uuid/v4'); //generates cryptographically strong secret with UUID-v4
-const oneHour = 3600000;
-const store = new mongoStore({modelName : "sessionName"} /* other store options here*/);
-app.use(session(
-{
-secret : uuidV4(),
-cookie : {maxAge: oneHour, secure: true},
-store : store,
-rolling : false,
-saveUninitialized : false,
-resave : false
+app.use(session({
+    secret: uuidV4(),
+    cookie: {
+        maxAge: 3600000, // one hour
+        secure: app.get('env') === 'production', // sets to true when prod
```

Comment:
This was the fix. Adding `app.get('env') === 'production'` will result in false when we're in a development mode (which we want as we're not using a secure connection).. and when running in production it will require a secure https connection.